src/include/miopen/find_solution.hpp

```diff
@@ -82,6 +82,17 @@ auto FindSolutionImpl(
                 MIOPEN_LOG_WE("Invalid config loaded from Perf Db: "
                               << s.SolverDbId() << ": " << config << ". Performance may degrade.");
             }
+            else if(!s.AltSolverDbId().empty() && db.Load(context, s.AltSolverDbId(), config))
+            {
+                MIOPEN_LOG_I("Perf Db: alternate record loaded: " << s.AltSolverDbId());
```

Comment:
```suggestion
                MIOPEN_LOG_I2("Perf Db: alternate record loaded: " << s.AltSolverDbId());
```

Reply:
I suggest we keep the higher log level, so that the developer is aware that such a heuristic is functioning